studio/schemas/carpark.js

```diff
@@ -15,6 +15,11 @@ export default {
       title: 'Slug',
       type: 'localeSlug',
     },
+    {
+      name: 'address',
```

Comment:
Suggest to add a new schema `coordinates` with data type `geopoint`, it will be useful for the search box
https://www.sanity.io/docs/geopoint-type